web-client/public/js/update-app.js

```diff
@@ -309,7 +309,14 @@ const hasExpressionData = (sheets) => {
 
     // renderNodeColoring: function () { }, // defined in graph.js
 
+const clearDropdownMenus = () => {
+    $(TOP_DATASET_SELECTION_SIDEBAR).html("");
+    $(BOTTOM_DATASET_SELECTION_SIDEBAR).html("");
```

Comment:
The original version used `empty` instead of `html("")` —does that make a difference?

Reply:
yes, it seems that empty was not replacing the HTML, and was fairly slow. This is much faster, and guarantees that the menu will clear.